proposal/WarClans612/proposal.rst

```diff
@@ -0,0 +1,171 @@
+
+Text content is unstructured and complex information that is challenging to search and index,
+therefore, by using tag information accompanied with the text content, we could extract main idea
+and topic within the unstructured information.
+
+Problem to solve
+================
+
+Industry Problem
+-----------------
+
+Nowadays, thanks to the development of social media and internet, more information can be easily
+accessible from anywhere. However, more problem arises as information flow faster, which overwhelm
+manual labor for many tasks. Most of the information in the web is in the shape if text content that
+by itself is hard to be digested by computer due to misaligned language used.
+
+Social media, content provider, modern encyclopedia (e.g. Wikipedia) consists of many content which 
+can be grouped into bags of topic. Using manual labor to separate and tag each piece of pages usually 
+yield the best results in correctness and precision of the tagged results. This approach cannot keep 
+up with the growth pace of the internet itself, so we would like to make an automatic approach for
+this problem.
+
+Tagging generation can be described simpler as structured information extraction from unstructured
+information (such as text) to enable easier digestion of information and analysis.
+
+Math Behind the Problem
+-----------------
+
+As we know that currently there is abundance of data in the internet which is good news for
+Machine Learning approach in computer science. Machine Learning approach requires enormous amount 
+of data to be fed into the system to let the system *learn* the correct approximation for the
+wanted results.
+
+Deep Learning is one of the branch in Machine Learning Approach which provides good results.
+It uses artificial neural network to do information processing and distributed communication
+which were inspired by biological systems. 
+
+In this project, we will use Deep Learning Approach that is usually powered by Back Propagation
+Algorithm (solved numerically) to *learn* the best function to approximate the results.
+
+Numerical Method to be Applied
+-----------------
+
+The main idea is to use past text and tag pairs to learn the intrinsic information and 
+perform tag generation for future text content. If possible, the newly generated text will
+then be supervised by human to ensure its validity for further model training.
+
+BackPropagation is the backbone for how the Deep Learning Approach could *learn* from 
+existing information. In fitting the neural network, BackPropagation will compute gradient of
+loss function with respect to the weights of the network for every input output pairs.
+
+By numerically optimizing the Neural Network (can be think as complex mathematical functions),
+the model could learn to minimize error towards the real expected output.
+
+Algorithm to be Used
+-----------------
+Extreme Deep Learning Classification refers to the problem of assigning to each document
```

Comment:
There should be a blank line separating the section title and the body.

Reply:
Solved in this 7959d31 commit